db/compaction/compaction_picker.cc

```diff
@@ -957,6 +960,11 @@ Status CompactionPicker::SanitizeCompactionInputFilesForAllLevels(
       }
     }
   }
+  if (!unchecked_input_files.empty()) {
+    return Status::Aborted(
```

Comment:
Should it be `InvalidArgument`? Also we iterate over all the `input_files` later, is it easier to check their level against `output_level` there? https://github.com/facebook/rocksdb/blob/ec346da98c6326f65473331cec06818a6fa3d23b/db/compaction/compaction_picker.cc#L1005

Reply:
Yeah, thanks, that a better place to check that. Updated.